packages/backend/src/graphql/invoice/mutations/invoiceUpdateMutationField.ts

```diff
@@ -39,7 +42,7 @@ builder.mutationField('invoiceUpdate', (t) =>
           invoiceWorkFrom: invoiceWorkFrom ?? undefined,
           invoiceWorkUntil: invoiceWorkUntil ?? undefined,
         },
-        where: { id: id.toString() },
+        where: { id: invoiceId.toString() },
```

Comment:
Let's keep invoice Id as `id` to be in line with the rest of codebase?
```suggestion
        where: { id: id.toString() },
```